README.md

```diff
@@ -403,43 +402,44 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Metadata Remover**](https://github.com/Crazy-Marvin/MetadataRemover) <sup>**[[F-Droid](https://f-droid.org/packages/rocks.poopjournal.metadataremover)]**</sup>
 * [**PrivacyBlur**](https://github.com/MATHEMA-GmbH/privacyblur) <sup>**[[F-Droid](https://f-droid.org/packages/de.mathema.privacyblur)]**</sup>
 * [**Scrambled Exif**](https://gitlab.com/juanitobananas/scrambled-exif) <sup>**[[F-Droid](https://f-droid.org/packages/com.jarsilio.android.scrambledeggsif)]**</sup>
 
 ### • Image Viewer & Gallery
 
-* [**Aves**](https://github.com/deckerst/aves) <sup>**[[F-Droid](https://f-droid.org/packages/deckers.thibault.aves.libre)]**</sup>
+* [**Aves**](https://github.com/deckerst/aves) <sup>**[[F-Droid](https://f-droid.org/packages/deckers.thibault.aves.libre)]**</sup> <sup>**[[IzzyOnDroid](https://apt.izzysoft.de/packages/deckers.thibault.aves)]**</sup>
 * [**Ente Photos**](https://github.com/ente-io/ente) <sup>**[[F-Droid](https://f-droid.org/packages/io.ente.photos.fdroid)]**</sup>
-* [**Fossify Gallery**](https://github.com/FossifyOrg/Gallery) <sup>**[[F-Droid](https://f-droid.org/packages/org.fossify.gallery)]**</sup>
+* [**Fossify Gallery**](https://github.com/FossifyOrg/Gallery) <sup>**[[F-Droid](https://f-droid.org/packages/org.fossify.gallery)]**</sup> <sup>**[[IzzyOnDroid](https://apt.izzysoft.de/packages/org.fossify.gallery)]**</sup>
 * [**Gallery**](https://github.com/IacobIonut01/Gallery) <sup>**[[F-Droid](https://f-droid.org/packages/com.dot.gallery)]**</sup>
 * [**Stingle Photos**](https://github.com/stingle/stingle-photos-android) <sup>**[[F-Droid](https://f-droid.org/packages/org.stingle.photos)]**</sup>
 
 ### • Keyboard
 
-* [**8VIM**](https://github.com/flide/8VIM) <sup>**[[F-Droid](https://f-droid.org/packages/inc.flide.vi8)]**</sup>
+* [**8VIM**](https://github.com/flide/8VIM) <sup>**[[F-Droid](https://f-droid.org/packages/inc.flide.vi8)]**</sup> <sup>**[[IzzyOnDroid](https://apt.izzysoft.de/packages/inc.flide.vi8)]**</sup>
 * [**AnySoftKeyboard**](https://anysoftkeyboard.github.io/) <sup>**[[F-Droid](https://f-droid.org/packages/com.menny.android.anysoftkeyboard)]**</sup>
 * [**fcitx5-android**](https://github.com/fcitx5-android/fcitx5-android) <sup>**[[F-Droid](https://f-droid.org/packages/org.fcitx.fcitx5.android)]**</sup>
-* [**FlorisBoard**](https://github.com/florisboard/florisboard) <sup>**[[F-Droid](https://f-droid.org/packages/dev.patrickgold.florisboard)]**</sup>
-* [**Fossify Keyboard**](https://github.com/FossifyOrg/Keyboard) <sup>**[[F-Droid](https://f-droid.org/packages/org.fossify.keyboard)]**</sup>
+* [**FlorisBoard**](https://github.com/florisboard/florisboard) <sup>**[[F-Droid](https://f-droid.org/packages/dev.patrickgold.florisboard)]**</sup> <sup>**[[IzzyOnDroid](https://apt.izzysoft.de/packages/dev.patrickgold.florisboard)]**</sup>  
+    FlorisBoard Beta:  <sup>**[[IzzyOnDroid](https://apt.izzysoft.de/packages/dev.patrickgold.florisboard.beta)]**</sup>
```

Comment:
@IzzySoft @offa Do you think it's necessary to recommend beta versions to new users?
Other beta version apps example:
- [InviZible Pro beta](https://apt.izzysoft.de/fdroid/index/apk/pan.alexander.tordnscrypt) at IoD
- Fossify Beta versions at [IoD](https://apt.izzysoft.de/fdroid/index.php) and [FD](https://search.f-droid.org/?q=fossify&lang=en)
- [Thunderbird Beta](https://f-droid.org/en/packages/net.thunderbird.android.beta) at FD
- [Nextcloud Dev](https://f-droid.org/en/packages/com.nextcloud.android.beta/) at FD _Intended for users willing to test new features and report bugs if they occur._
etc...

Reply:
Fair question – so directly asked, I would say no.